pontoon/sync/vcs/models.py

```diff
@@ -177,7 +177,7 @@ def changed_locales_files(self):
 
                 log.info(
                     'Changed files in {} repository, all: {}'.format(
```

Comment:
Python newbie passing by: wouldn't it be enough to do `u'Changed files…`?

Reply:
That would do, too. But I'm trying to stay consistent with the rest of the code.

Here's how it works: Default string representation of the Project instance is [defined](https://github.com/mozilla/pontoon/blob/master/pontoon/base/models.py#L1059) as `self.name`. Note the [decorator](https://github.com/mozilla/pontoon/blob/master/pontoon/base/models.py#L1002) on top which makes the output `unicode`. Since `str` and `unicode` concatenation (which is what we do in this line) results in a `unicode` string, that's all we need.

HTH

Reply:
> HTH

Yep, thanks.